css/style.css

```diff
@@ -0,0 +1,146 @@
+/*============ Start Global ==============*/
+body{
+    color: #cccccc;
+    font-family: 'Philosopher', sans-serif;
+    text-align: left;
+}
+*{
+    margin: 0;
+    padding: 0;
+}
+.container{
+    width: 85%;
+    margin-left: auto;
+    margin-right: auto;
+}
+.body-content{
+    background-image: url('../images/containtBackground.png');
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+    background-repeat: no-repeat;
+    padding: 0 50px;
+    padding-bottom: 150px;
+}
+/*============ Start Header ==============*/
+.logo{
+    padding: 40px 0 50px 0;
+    font-size: 60px;
```

Comment:
instead of using **px** as your unit, use **em** or **rem**. This will make your CSS scale better in a responsive design. Google "em" and "rem" if you don't know them.